sched/Kconfig

```diff
@@ -600,6 +600,13 @@ config SCHED_USER_IDENTITY
 		Those can then be managed using the interfaces.  Child tasks will
 		inherit the UID and GID of its parent.
 
+config SCHED_THREAD_LOCAL
+	bool "Support __thread keyword"
+	default n
+	depends on ARCH_ARM
+	---help---
+		The keyword __thread can be used to declare thread local variables.
```

Comment:
Please tell more! I think it is important to explain the performance gain when using it. I wasn't aware of it until read here: https://stackoverflow.com/questions/32245103/how-does-the-gcc-thread-work